### ISSUE TITLE
Q: native JRE not bundle when executed on Jenkins slave

### ISSUE THREAD

**Issue by myron0815** [javafx-maven-plugin/javafx-maven-plugin#232]
TL;DR
executing in command line works fine
executing on jenkins slave doesn't bundle JRE

Background:
Having a linux buildserver.
Since OpenJDK doesn't go well with JavaFX, i added a Windows Jenkins slave with recent Oracle 1.8_101 as system service. Building with Jenkins works w/o any errors, but in /native/ directory there is no JRE bundled, and no *.dll files present.

On same slave, opened a CMD as Administrator and spawned a new one (with psexec) with SYSTEM privileges (= same account under witch Jenkins runs - as service)
"mvn clean install" just works fine, copying everything as expected?!?

Any ideas how to debug this?
What might be the difference?
Didn't see any debug output - but could you print the exact call to the Oracle packager somehow?

br

**Comment by FibreFoX** [comment]
First of all, this is something jenkins-related, not plugin-related (as it works fine without jenkins-node-setup). Second, OpenJDK goes well when haven OpenJFX installed too.

Please make sure your system is setup correctly, including the JAVA_HOME-environment (just in case it is missing). It is highly something with your system, not the plugin (I can't think about any way this would go wrong, probably something with file-permissions).

**Comment by myron0815** [comment]
Thx for your input; will check privileges.
java_home and everything seems to be correct.

Buildserver is not under my control... 
and i got said, that there is no openfx available (for this - bit dated - distro)

Anyway... thx
Will try to get some debug output from the bundler...

**Comment by myron0815** [comment]
root cause #237
The duplicated execution throws an 'access denied' error recopying the files...
